go/tunnels/id_generation.go

```diff
@@ -0,0 +1,24 @@
+package tunnels
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+)
+
+var nouns = []string{"pond", "hill", "mountain", "field", "fog", "ant", "dog", "cat", "rabbit", "shoe", "campsite", "plane", "sofa", "chair", "library", "book", "ocean", "lake", "river", "horse"}
```

Comment:
consider swapping out rabbit, campsite, sofa, and library.

Reply:
updated